src/pages/community-champions.js

```diff
@@ -470,6 +490,137 @@ const OtherDetailsAndRequirements = () => (
     </OtherDetailsAndRequirementsContainer>
 );
 
+const MeetTheChampionsContainer = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.greyDarkThree};
+    margin: ${size.xlarge} -${size.xxlarge};
+    padding: ${size.xlarge} ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        margin: ${size.large} -${size.default};
+        padding: ${size.large} ${size.default};
+    }
+`;
+
+const ChampionsContainer = styled('div')`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    row-gap: ${size.default};
+    @media ${screenSize.upToLarge} {
+        row-gap: ${size.xsmall};
+    } ;
+`;
+
+const ChampionItemContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    padding: ${size.default};
+    text-align: center;
+    width: ${CHAMPION_ITEM_CONTAINER_WIDTH};
+    @media ${screenSize.upToMedium} {
+        padding: ${size.default} 0;
+        width: ${CHAMPION_ITEM_CONTAINER_MOBILE_WIDTH};
+    }
+`;
+
+const LocationContainer = styled('div')`
+    display: flex;
+    justify-content: center;
+    margin-top: ${size.tiny};
+`;
+
+const PinLocationImage = styled('img')`
+    margin-right: ${size.tiny};
+`;
+
+const LocationText = styled(P4)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+`;
+
+const ChampionLocation = ({ location }) => (
+    <LocationContainer>
+        <PinLocationImage src={PinLocationIcon} />
+        <LocationText collapse>{location}</LocationText>
+    </LocationContainer>
+);
+
+const ChampionTitleText = styled(P3)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+`;
+
+const ChampionProfilePicture = styled(ProfileImage)`
+    margin-bottom: ${size.default};
+    div {
+        background-size: cover;
```

Comment:
![Screen Shot 2021-07-14 at 11 47 33 AM](https://user-images.githubusercontent.com/46455728/125652091-2b1b8e8b-1ccd-46f9-9e26-d2761a87fd10.png)

it's a bit hard to tell but i noticed the default image was repeating. i added this so that the image can be resized to cover the whole container